src/main/java/com/cgdp/library/service/AuthorService.java

```diff
@@ -21,4 +22,9 @@ public AuthorDTO createAuthor(String authorName) {
         AuthorEntity savedAuthor = authorRepository.save(authorEntity);
         return new AuthorDTO(savedAuthor.getId(), savedAuthor.getName());
     }
+
+    public AuthorEntity getById(Long authorId) {
+        return authorRepository.findById(authorId)
```

Comment:
It should be calling `getReferenceById` but this is not even being used so let's remove it